packages/notebook/src/windowing.ts

```diff
@@ -97,6 +99,30 @@ export class NotebookWindowedLayout extends WindowedLayout {
     }
   }
 
+  /**
+   * Returns Notebook's footer
+   *
+   * @return NotebookFooter | null
+   */
+
+  get footer(): NotebookFooter | null {
+    return this._footer;
+  }
+
+  /**
+   * Set Notebook's footer
+   *
+   * @param v: Widget | null
+   */
+  set footer(v: NotebookFooter | null) {
```

Comment:
Hi @andrii-i , the UI of the rectangle should be aligned with the cells, as I commented above. 

However, I do not think that it should be there permanently. I agree that for a mobile use this is not efficient but for the moment I think we should focus on the web-oriented problem and tackle that later.

Reply:
@GabrielaVives  thank you for the feedback. I'd like to better understand your point of view, could you please elaborate on both points (why UI element should not be visible permanently; why we should focus on web-oriented problems only)?

Reply:
JupyterLab is working well on mobile and are people who use it there on daily basis. Therefore I think we should make sure any design decision / UI element works well on mobile.

Reply:
Hi @andrii-i , yes of course! First, let's agree that this feature _could_ be visible permanently if it's encouraged by a specific user need, for the moment with the information I have about the needs from which this solution came from, I advise **not to** for the following reasons:
- in the case of the notebook being small, this button will be visible all the time. This is not a problem for buttons in the toolbar for example as it is specifically meant for this, but the area of the notebook is meant for code, images, graphs, content created by the user. Having a (big) button in this area will create visual noise and bring an unnecessary cognitive load to the user;
- in the case of larger notebooks, this will also happen when the user is at the bottom of it, which probably represents most of the time the user is creating content;
- from benchmarking, we can see that this feature already exists in similar applications, such as Colab or VScode. The button to add a cell appears on hover in both of these. Even though we must question these choices and by no means take decisions based only on a benchmark and "they did it so it must be good", we must take into account that this reflects our user's mental models.

I hope this answered your question, please tell me if you would like to discuss more about this topic!

Reply:
To support both scenario we could add a conditional style depending if the device support hover or not ([MDN doc](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/hover)). This is for example what is done for tuning the behavior the file browser check boxes:

https://github.com/jupyterlab/jupyterlab/blob/a51b7d9df2b4e24d39b2333e22f46a96d004bfa6/packages/filebrowser/style/base.css#L243

Reply:
@GabrielaVives Thank you for providing detailed reply. These are really good points, I especially agree that it is important to not have notebook visually cluttered. 

I think @fcollonval's idea might be a good middle ground between absence or visual clutter and usability on mobile. Again, I don't use jupyterlab on mobile much so people who do like @tonyfast could give a better insight if always showing "add cell" button on mobile (vs showing it on touch / mouse over only) would be useful. Just wanted to took both options into consideration.

Reply:
on a small mobile screen, the cell toolbar is always persistent so i can always add a cell. i think since there is already interaction for adding a new cell in the toolbar maybe we don't need anything special for very small screens. 

adding more feature to a small screen might be bad, but on a larger tablet this UX might be possible because there is more screen real estate.